util/test_timeline.py

```diff
@@ -224,9 +252,7 @@ def test_match(event, entry):
         elif entry["special_type"] == "applydebuff" and event.startswith(entry["special_line"]):
             # Matching this format generically:
             # |Dadaluma Simulation|0.00|E0000000||4000AE96|Guardian
-            buff_match = re.search(
-                "\|{}\|([^\|]*\|){{4}}{}".format(entry["buff_name"], entry["buff_target"]), event
-            )
+            buff_match = re.search(entry["regex"], event, re.IGNORECASE)
```

Comment:
*This* little annoyance right here was what made `gainsEffect` syncs perpetually fail. We can't rely on the casing of the `effectId` in the timeline file to match that in the network log. ~~It took me "only" half an hour of "WHY WON'T THIS WORK? IT MATCHES!" to debug~~

Reply:
Oh nooo.

If it makes you feel any better, everything is uppercase in the alpha plugin (but we probably need to keep this indefinitely to support old network log files).